python/servo/post_build_commands.py

```diff
@@ -81,7 +81,7 @@ class PostBuildCommands(CommandBase):
     def run(self, params, release=False, dev=False, android=None, debug=False, debugger=None,
             headless=False, software=False, bin=None, emulator=False, usb=False, nightly=None):
         env = self.build_env()
-        env["RUST_BACKTRACE"] = "1"
+        env["RUST_BACKTRACE"] = env["RUST_BACKTRACE"] if 'RUST_BACKTRACE' in env else "1"
```

Comment:
If we use `env.get("RUST_BACKTRACE", "1")` I think this will be more readable (same for other uses).

Reply:
Yeah!! This looks much better! Let me update! Thanks :D